tests/test_html2docx.py

```diff
@@ -39,6 +40,8 @@ def test_html2docx(html_path, spec_path):
         assert p.style.name == p_spec.get("style", "Normal")
         if p.alignment:
             assert int(p.alignment) == p_spec["alignment"]
+        if p.paragraph_format.left_indent:
```

Comment:
If the spec mentions `left_indent` but the paragraph does not have it, the test will pass.